src/utils/logging.ts

```diff
@@ -0,0 +1,63 @@
+import path from 'path';
+import { configure, addLayout, Configuration, getLogger, LoggingEvent } from 'log4js';
+import { RequestLogName } from '../middleware/logging';
+
+const logLocation =
+    process.env.NODE_ENV === 'production'
+        ? '/var/log/dotcom-components/dotcom-components.log'
+        : `${path.resolve('logs')}/dotcom-components.log`;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const logFields = (logEvent: LoggingEvent): any => {
+    const fields = {
+        stack: 'support',
+        app: 'dotcom-components',
+        stage: 'CODE',
+        '@timestamp': logEvent.startTime,
+        '@version': 1,
+        level: logEvent.level.levelStr,
+        level_value: logEvent.level.level,
+    };
+
+    if (logEvent.data[0] === RequestLogName && typeof logEvent.data[1] === 'object') {
```

Comment:
This isn't ideal but we couldn't find a neater way of doing this with log4js.
`logEvent.data` is the array of arguments passed to `logger.info(...)`. In the request logging middleware we make the first argument 'request', to indicate that the data should be added at the top level of the log event